src/component-library/utils/prop-types.ts

```diff
@@ -23,6 +23,10 @@ export const alignItems = tuple('flex-start', 'center', 'flex-end', 'stretch', '
 
 export const normalAlignments = tuple('start', 'center', 'end');
 
+export const direction = tuple('row', 'row-reverse', 'column', 'column-reverse');
```

Comment:
Minor, but since all data types are string literals, what's the benefit of using a tuple here?